File: WDI Exercises/sinatras_songs/artist.rb
class Artist
  attr_accessor :name, :genre, :albums
  #attr_reader :name, :genre, :albums
  #attr_writer :name, :genre, :albums
  def initialize(name, genre, albums)
    @name = name
    @genre = genre
    @albums = albums
  end
end

porter_robinson = Artist.new('Porter Robinson', 'Complextro', 'Worlds, Language, Spitfire')

File: WDI Exercises/sinatras_songs/Artist.js
function Artist(name, genre, albumsArray) {

  this.props = {};
  this.props.name = name;
  this.props.genre = genre;
  this.props.albums = albumsArray;

  this.toObject() {
    return this.props;
  }

}

var sinatra = new Artist('Frank Sinatra', 'Classic', 'Youtube');
